### DISCUSSION TITLE
Debugging of BearingFluidFlow (from 1.4.2 to 1.5.0)

### DISCUSSION THREAD

**Discussion by Vibro49** [petrobras/ross#1120] [Q&A] [answered]
Dear Community,

I would like to ask for your help with a debugging task. Unfortunately, I'm not very experienced in Python, but I had a code that was working well back in July, likely with ROSS version 1.4.2. I’ve attached the link of my file, which includes the results: https://colab.research.google.com/drive/1mak0liWsCVlhNyxNm_0ebGxcgjurcbo_?usp=sharing
Now, as I'm working in Google Colab with ROSS version 1.5.0, my previously functional code is no longer working. I'm encountering an issue that might be related to the "nz" parameter in the BearingFluidFlow class, though I believe it is defined in the code.
I would greatly appreciate any assistance or suggestions to resolve this issue.

Thank you in advance for your help!

**Comment by ViniciusTxc3** [comment, marked as answer]
Hi @Vibro49, I look at your code and this issue occurred in PR but was recently fixed in PR #1102. It will be updated in the next version of ross 1.5.1. To make your code run at this time, you can import the library as follows:

`pip install ross-rotordynamics`
to
`!pip install git+https://github.com/petrobras/ross.git`

This will install the latest version of ross from github. I tested your code with this change and it worked.

**Reply by Vibro49** [reply]
Dear @ViniciusTxc3,
Yes it works now, Thank you very much!
